build

```diff
@@ -37,6 +37,7 @@ case 'full':
 	execSync(`node ./build-tools/build-indexes`, options);
 	execSync(`node ./build-tools/build-learnsets`, options);
 	execSync(`node ./build-tools/build-minidex`, options);
+	execSync(`node ./build-tools/build-sets`, options);
```

Comment:
I would not add this to the full build until after we started using it, I think.

Reply:
Done. TBH, `npm install` is pretty slow (seems to also do dependency checks on other things in the package.json instead of just install? I wonder if I can turn that off), so maybe we'll always want to update sets piecemeal

Reply:
Found it. `--no-audit`. And `--force` to always go remote

Reply:
Hmm
maybe I dont want force